documentation/content/codingstyle.rst

```diff
@@ -167,61 +167,44 @@ For the function returning we try to respect the errno codes, for example::
 As you can notice from the examples, we use snake case for function
 definitions, this is valid for all the kw code.
 
-Redirection
------------
-
-Redirection operators should be written with space before, but no space after
-them. In other words, write ``printf '%s\n' test >"$file"`` instead of
-``printf '%s\n' test> "$file"`` or ``printf '%s\n' test > "$file"``. Note that
-even though it is not required by POSIX to double-quote the redirection target
-in a variable (as shown above), our code does so because some versions of bash
-issue a warning without the quotes::
-
-    (incorrect)
-    cat hello > world < universe
-    printf '%s\n' hello >$world
-
-    (correct)
-    cat hello >world "$world"
-
 Command substitution and arithmetic expression
 ----------------------------------------------
 
 We prefer `$( ... )` for command substitution; unlike \`\`, it properly nests.
-It should have been the way Bourne spelled it from day one, but unfortunately
-isn't.
 
 When using command substitution to access the contents of a file the cat
-command (``$(cat <file>)``) can be replaced with a < which is equivalent but
+command (``$(cat <file>)``) can be replaced with a `<` which is equivalent but
 faster (``$(< <file>)``). E.g.: ``$(cat "$file") => $(< "$file")``
 
-For arithmetic expansion we use `$(( ... ))`.
+For arithmetic expansion we use `(( ... ))`.
 
 Check for command
 -----------------
 
-If you want to find out if a command is available on the user's
-$PATH, you should use 'type ', instead of 'which '.
-The output of 'which' is not machine parsable and its exit code
-is not reliable across platforms.
+If you want to find out if a command is available on the user's `$PATH`, you
+should use the function `command_exists()` available under kw lib. If you are
+working in a plugin or have a strong reason not to use `command_exists()`, you
+should use `command` instead of `which` since the letter is not machine
+parsable and its exit code is not reliable across platforms.
 
 How to include/import files
 ---------------------------
-Do not source code using `.` or `source`. We have a helper function for that
-named `include` in `kw_include.sh` and it should be used any and everytime a
-file needs to be sourced, `. file.sh --source-only` should only be used to
-source `include.sh` itself. The `include` function guarantees us that no file
-will be sourced twice, making the kw dev life easier with one thing less to
-worry about.
-
-Test name
----------
+
+Do not source code using `.` or `source` unless you have a very strong
+argument. We have a helper function for that named `include` in `kw_include.sh`
+and it should be used any and every time a file needs to be sourced, `. file.sh
+--source-only` should only be used to source `include.sh` itself. The `include`
+function guarantees us that no file will be sourced twice, making the kw dev
+life easier with one thing less to worry about.
```

Comment:
Double back ticks for command arguments and functions, keep the single back ticks for file names.